src/sentry/integrations/utils/atlassian_connect.py

```diff
@@ -28,10 +30,10 @@ def get_query_hash(
     for k, v in sorted(query_params.items()):
         # don't include jwt query param
         if k != "jwt":
-            if isinstance(v, list):
```

Comment:
I'm seeing an inversion of the conditionals but nothing else. What's the reason for the change? Is checking for a string first better in some way?

Reply:
the type annotation is `str | Sequence[str]` and a narrowing via `isinstance(..., list)` doesn't actually narrow anything because not all `Sequence`s are `list`s